### ISSUE TITLE
fix search: "pie" vs "pies" etc.

### ISSUE THREAD

**Issue by karindalziel** [CDRH/austen#61]
I double checked , and this is not a problem with other sites (see cody).

**Comment by karindalziel** [comment]
OK! I have done research on this and it looks like we've just happened to hit upon a really weird problem. 

If you search for "pie" in austen you get no results. However, I checked in cody, and it brings up results there. 

Results for cody:

- pies - 13
- pie - 9
- pi - 13

It appears that "pies" is stemming to "pi" while "pie" is not stemming to anything (including "pies"). 

http://cors1601.unl.edu:8983/solr/austen/select?q=text%3Api&wt=json&indent=true&hl=true&hl.fl=text&hl.simple.pre=%3Cem%3E&hl.simple.post=%3C%2Fem%3E

Anyway, closing this out, but hopefully we remember in case it comes up again. we may want to research how to add custom stemming in case we ever need to build a recipe site.
